packages/synthetic-chain/public/upgrade-test-scripts/env_setup.sh

```diff
@@ -201,7 +201,7 @@ voteLatestProposalAndWait() {
 
 printKeys() {
   echo "========== GOVERNANCE KEYS =========="
-  for i in ~/.agoric/*.key; do
+  for i in /usr/src/upgrade-test-scripts/keys_for_test_only/*.key; do
```

Comment:
consider making a README.md in `keys_for_test_only` to explain where they came from and why they're static